roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -69,23 +69,19 @@
     namespace: "{{ cr_info.metadata.namespace }}"
     merge_type: ['merge', 'json']
     resource_definition: "{{ old_cr_templates | k8s_inject_ownership(cr_info) }}"
-  when: "{{ (old_cr_templates is defined) and (old_cr_templates | length > 0) }}"
+  when: (old_cr_templates is defined) and (old_cr_templates | length > 0)
 
 - name: "Count all new templates in file"
   set_fact:
     new_templates: "{{ lookup('file', 'common-templates-'+ version +'.yaml').split('\n---\n') | select('search', '(^|\n)[^#]') | list | length }}"
 
-- name: "Set label"
-  set_fact:
-    label: "template.kubevirt.io/version={{ version }}"
-
 - name: "Get all templates"
   set_fact:
-    deployed_templates_after: "{{ lookup('k8s', api_version=ct_status.result.results[0].result.apiVersion, kind='template', label_selector=label) | length }}"
+    deployed_templates_after: "{{ lookup('k8s', api_version=ct_status.result.results[0].result.apiVersion, kind='template', label_selector='template.kubevirt.io/type=base') | length }}"
 
 - name: "Set Available status"
   set_fact:
-    available: "{{ true if new_templates <= deployed_templates_after else false }}"
+    available: "{{ True if new_templates <= deployed_templates_after else False }}"
```

Comment:
Ansible doesn't particularly care about True vs true, but yaml linter may. Why the change?

Reply:
Just for consistency really, nothing more

Reply:
for a layperson like myself, it is important to keep cosmetics like this on another PR.